appmetrics.properties

```diff
@@ -16,3 +16,7 @@ com.ibm.diagnostics.healthcenter.logging.plugins=info
 
 # Start with method profiling enabled/disabled: on | off
 com.ibm.diagnostics.healthcenter.data.profiling=off
+
+# Allow probes to transmit data to the client: true | false
+# Defaults to false - probe data not sent.
+#com.ibm.diagnostics.healthcenter.probes.transmit=false
```

Comment:
@seabaylea  should we still be naming this com.ibm.diagnostics.healthcenter.probes.transmit?  This is purely for the appmetrics plugins for the agent so I would think "appmetrics.probes.transmit" would be a better names